.gitmodules

```diff
@@ -1,6 +1,3 @@
 [submodule "native"]
 	path = native
 	url = git@github.com:fishpondstudio/IndustryIdle-Native.git
-[submodule "assets/Script/Config"]
```

Comment:
This probably should not be checked in?

Reply:
Sorry, not sure how this got checked in.